src/main/resources/application-prod.yml

```
@@ -0,0 +1,9 @@
+# 프로덕션 서버환경 전용 설정
+app:
+  oauth2:
+    authorized-redirect-uri: ${FRONTEND_URL}/oauth2/redirect  # 프론트엔드로 리다이렉트
+  cookie:
+    domain: ${COOKIE_DOMAIN:.coffeechat.kro.kr}  # 프로덕션 도메인 설정
+    same-site: ${COOKIE_SAMESITE:None}  # HTTPS 환경에서 None 사용
+    secure: ${COOKIE_SECURE:true}
+    http-only: ${COOKIE_HTTPONLY:true}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)